pkg/question/question.go

```diff
@@ -1197,10 +1200,28 @@ func AskTerminationConfirmation(instanceIds []string) string {
 	return answer
 }
 
-func AskCapacityType() string {
-	question := fmt.Sprintf("Select capacity type. Spot instances are available at up to a 90%% discount compared to On-Demand instances,\nbut they may get interrupted by EC2 with a 2-minute warning")
+func AskCapacityType(instanceType string) string {
+	ec2Pricing := ec2pricing.New(session.New())
+	onDemandPrice, err := ec2Pricing.GetOnDemandInstanceTypeCost(instanceType)
+	formattedOnDemandPrice := ""
+	if err == nil {
+		onDemandPrice = math.Round(onDemandPrice*10000) / 10000
+		formattedOnDemandPrice = fmt.Sprintf("($%s/hr)", strconv.FormatFloat(onDemandPrice, 'f', -1, 64))
```

Comment:
I didn't do that because that could lead to an output like $0.0200. I thought this method allows for rounding and truncating whereas that method is only for cutting off remaining numbers after a certain decimal point

Reply:
For instance this is for t1.micro:
1. On-Demand($0.02/hr)
2. Spot($0.0046/hr)

Reply:
Okay, that makes sense. The `-1` as the third parameter of `FormatFloat()` means "use as many decimal places as necessary, but no more", so combined with rounding to 4 significant figures on the previous line, that accomplishes what we want without any trailing zeroes.

This looks good to me! Thanks for the explanations.